Aptoide/aptoide-uploader
# Bugfix/mob 306 backgroud service upload

**What does this PR do?**

   Allows uploads to be done in the background after the application is closed

**Where should the reviewer start?**

- NotificationApplicationView.java
- UploadManager.java
- NotificationPresenter.java
- MemoryDraftPersistence.java

**How should this be manually tested?**

 Open Uploader, start an upload, close the application process after the notification shows up and check if the upload progress continues

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/MOB-306

**Code Review Checklist**
